apps/desktop-wallet/src/api/queries/tokenQueries.ts

```diff
@@ -63,7 +63,7 @@ export const fungibleTokenMetadataQuery = ({ id, skip }: TokenQueryProps) =>
       ? async () => {
           const tokenMetadata = await fungibleTokenMetadataBatchFetcher.fetch(id)
 
-          return tokenMetadata ? convertDecimalsToNumber(tokenMetadata) : undefined
+          return tokenMetadata ? (convertDecimalsToNumber(tokenMetadata) as UnlistedFT) : undefined
```

Comment:
Again, dev experience, I find it nicer to see `UnlistedFT` than an object with all its properties, especially when it is used in combined types like `ListedFT | UnlistedFT | NFT | ...`

Reply:
Okay. I had issue in the past with casting. There were some mistakes (fields not defined) that were not catch by the compiler because of this. That's why I'm kinda hesitant now.

Reply:
I think you've got a very good point here. We should probably prefer to define return types instead of casting. I've updated the code in this commit to reflect that. WDYT of this change?  d67ef36cf